download/index.html

```diff
@@ -1,20 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<!-- Redirect page
-made with <3 by RDIL for Sk1er
```

Comment:
how dare you remove my credit

Reply:
probably increases the download time of the document for redirect, but if you insist..

Reply:
comments aren't downloaded by the majority of most modern browsers

Reply:
@RDIL comments are loaded with the page